functions/Get-DbaLastBackup.ps1

```diff
@@ -0,0 +1,158 @@
+		$exclude = $psboundparameters.Exclude
+	}
+
+	PROCESS
+	{
+		foreach ($servername in $SqlServer)
+		{
+			try
+			{
+				$server = Connect-SqlServer -SqlServer $servername -SqlCredential $Credential
+                # setting defaultinitfields for performance, tip from WSMelton?, if this is in the connect-sqlserver function, it can be removed here
+                $server.SetDefaultInitFields($true)
+                $server.SetDefaultInitFields([Microsoft.SqlServer.Management.Smo.DataFile], $false)
+			}
+			catch
+			{
+				if ($SqlServer.count -eq 1)
+				{
+					throw $_
+				}
+				else
+				{
+					Write-Warning "Can't connect to $servername. Moving on."
+					Continue
+				}
+			}
+
+			$dbs = $server.Databases | Where-Object { $_.name -ne 'TempDb' }
+
+			if ($databases.count -gt 0)
+			{
+				$dbs = $dbs | Where-Object { $databases -contains $_.Name }
+			}
+
+			if ($exclude.count -gt 0)
+			{
+				$dbs = $dbs | Where-Object { $exclude -notcontains $_.Name }
+			}
+
+
+			foreach ($db in $dbs)
+			{
+                $obj = $null
+				Write-Verbose "Processing $($db.name) on $servername"
+
+				if ($db.IsAccessible -eq $false)
+				{
+					Write-Warning "The database $($db.name) on server $servername is not accessible. Skipping database."
+					Continue
+				}
+                # To avoid complicated manipulations on datetimes depending on locale settings and culture,
+                # dates are compared to 0, which represents 1/01/0001 0:00:00
+				$TimeSinceFullBU = if ($db.LastBackupdate -eq 0) {""} else {(New-TimeSpan -Start $db.LastBackupdate).Tostring()}
+                $TimeSinceFullBU = if ($db.LastBackupdate -eq 0) {""} else {$TimeSinceFullBU.split('.')[0..($TimeSinceFullBU.split('.').count - 2)] -join ' days ' }
+
+				$TimeSinceDiffBU = if ($db.LastDifferentialBackupDate -eq 0) {""} else {(New-TimeSpan -Start $db.LastDifferentialBackupDate).Tostring()}
+                $TimeSinceDiffBU = if ($db.LastDifferentialBackupDate -eq 0) {""} else {$TimeSinceDiffBU.split('.')[0..($TimeSinceDiffBU.split('.').count - 2)] -join ' days ' }
+
+				$TimeSinceLogBU = if ($db.LastLogBackupDate -eq 0) {""} else {(New-TimeSpan -Start $db.LastLogBackupDate).Tostring()}
+                $TimeSinceLogBU = if ($db.LastLogBackupDate -eq 0) {""} else {$TimeSinceLogBU.split('.')[0..($TimeSinceLogBU.split('.').count - 2)] -join ' days ' }
+
+				$daysSinceDbCreated = (New-TimeSpan -Start $db.createDate).Days
+
+                If ($daysSinceDbCreated -lt 1 -and $db.LastBackupDate -eq 0) { $Status = 'New database, not backed up yet' }
+				elseif ((New-TimeSpan -Start $db.LastBackupDate).Days -gt 0 -and (New-TimeSpan -Start $db.LastDifferentialBackupDate).Days -gt 0){$Status = 'No Full or Diff Back Up in the last day'}
+				elseif ($db.RecoveryModel -eq "Full" -and (New-TimeSpan -Start $db.LastLogBackupDate).Hours -gt 0){$Status = 'No Log Back Up in the last hour'}
+				else { $Status = 'OK' }
+				
+				$obj = [PSCustomObject]@{
```

Comment:
$result is good enough, actually. I'm just not partial to obj in any form (other than $object) because it's very VBScripty.

I do really really like how you returned the object in the body, so there's no collection and waiting. That will be something we'll have to change in all of our other commands for 1.0